src/utils/cmds/putval.c

```diff
@@ -187,22 +185,30 @@ cmd_status_t cmd_parse_putval(size_t argc, char **argv,
   for (size_t i = 1; i < argc; ++i) {
     value_list_t *tmp;
 
+    /* Copy option because cmd_parse_option modifies its argument. */
+    char *option = sstrdup(argv[i]);
     char *key = NULL;
     char *value = NULL;
 
-    status = cmd_parse_option(argv[i], &key, &value, err);
+    status = cmd_parse_option(option, &key, &value, err);
     if (status == CMD_OK) {
       int option_err;
 
       assert(key != NULL);
       assert(value != NULL);
       option_err = set_option(&vl, key, value, err);
+      key = NULL;
+      value = NULL;
```

Comment:
Why?  These are are internal to loop, set to NULL at start of loop, and not used before looping again?